app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  has_one :posts
+  has_one :users
```

Comment:
- [ ] Same thing here, The `Like` belongs to the users and posts. So the right relationship here would be `belongs_to` rather than `has_one`. 
Let us try and make these changes.